src/components/client/home/order/qr/NexQrAccept.tsx

```diff
@@ -1,9 +1,9 @@
 const productqr = {
-  title: 'Validado',
-  name: 'Validado',
+  title: 'Validated',
+  name: 'Validated',
   img: '../../../public/images/caja/check.png'
 }
-const NEXOrderSideBarQrAccept = () => {
+const NexQrAccept = () => {
   return (
     <div>
       <h3 className="newsletter-title-qr">{productqr.title}</h3>
```

Comment:
siguiendo el uso de camel case, renombra a `productQr.title`
```suggestion
      <h3 className="newsletter-title-qr">{productQr.title}</h3>
```